omnibus/config/software/datadog-agent-finalize-licenses.rb

```diff
@@ -0,0 +1,17 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-present Datadog, Inc.
+
+name "datadog-agent-finalize-licenses"
+description "steps required to remove licenses"
+skip_transitive_dependency_licensing true
+always_build true
+
+build do
+  # compress separately each license file in the install dir
```

Comment:
Does this help that much?
ISTM that putting "#{install_dir}/LICENSES/* into a single compressed tarball would give the most compression. 
Or does that violate the spirit of providing the licenses?

Reply:
As discussed IRL, it was advised by the opensource team